pkg/operational/health.go

```diff
@@ -45,8 +43,7 @@ func (hs *Server) Serve() {
 func NewHealthServer(opts *config.Options, isAlive healthcheck.Check, isReady healthcheck.Check) *Server {
 
 	handler := healthcheck.NewHandler()
-	address := net.JoinHostPort(defaultServerHost, opts.Health.Port)
-
+	address := net.JoinHostPort(opts.Health.Address, opts.Health.Port)
```

Comment:
The health metrics are used also by others in existing deployments, so we need to maintain that 0.0.0.0 is the default, even if the user did not specify it.

Reply:
Verify that the health_test.go runs successfully. We also need to (eventually) add a test to verify that the new functionality (address) is handled properly.